code/Modules/Props/Types/CustomProp.cs

```diff
@@ -1,19 +1,43 @@
 ﻿using Sandbox;
 using ssl.Modules.Items.Carriables;
+using ssl.Modules.Props.Data;
 using ssl.Modules.Selection;
 using ssl.Player;
 
-namespace ssl.Modules.Props
+namespace ssl.Modules.Props.Types
 {
     /// <summary>
     /// A prop is an object not in inventory
     /// Inspired by sandbox Props
     /// </summary>
-    public abstract partial class Prop : BasePhysics, ISelectable
+    public partial class CustomProp : BasePhysics, ISelectable
```

Comment:
We can use Prop as the class name, we'll just need to make an alias. Prop is cleaner than CustomProp